lib/config.js

```diff
@@ -6,15 +6,15 @@ import {config} from 'bedrock-web';
 // set bedrock-web-wallet default config
 config.wallet = {
   defaults: {
-    edvBaseUrl: `https://localhost:55443/edvs`,
+    edvBaseUrl: 'https://localhost:55443/edvs',
     // FIXME: do not use `referenceId` and put that information on the server,
     // rather have profile agent information store a location to read EDV IDs
     // from based on local names; remove this from the shared web app config
     edvs: {
-      users: _getReferenceId('users'),
-      credentials: _getReferenceId('credentials')
+      users: encodeURIComponent('localhost:users'),
+      credentials: encodeURIComponent('localhost:credentials')
```

Comment:
This still does not match what it was doing before.

Reply:
Fixed here [f73be53](https://github.com/digitalbazaar/bedrock-web-wallet/pull/4/commits/f73be5310cdd2a0d1d7703dd2c073c7add8522ff)